x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/io/stream/PlanNamedTypes.java

```diff
@@ -192,8 +194,6 @@
 import org.elasticsearch.xpack.ql.expression.predicate.regex.WildcardPattern;
 import org.elasticsearch.xpack.ql.index.EsIndex;
 import org.elasticsearch.xpack.ql.options.EsSourceOptions;
-import org.elasticsearch.xpack.ql.plan.logical.Aggregate;
```

Comment:
That's a subtle change to serialisation! 

It's fine though of course, since the classes are identical.

Reply:
Thanks Alex. Yes, the internal structures of these classes are the same for now.